puppet-module-design.md

```diff
@@ -239,6 +240,13 @@ operating system and not site specific information allows the module to be
 reusable. Information relating to sites, environments or installations should be
 provided at the `Profile` layer.
 
+The lowest-priority default value should be provided as a parameter default
+in-class, rather than in data-in-modules common.yaml. This is to ensure that
+`puppet strings` generated documentation reveals that no default needs to be
```

Comment:
In addition to `puppet strings`, this also makes the default value appear in the PE Classifier.

Reply:
I didn't think about that, but it points out that there is more than one use case impacted. I don't think we need to mention PE specifically here but I've updated the copy to mention the utility to users reading the Puppet code, in addition to users reading `puppet strings` documentation.